src/NzbDrone.Core/IndexerSearch/Definitions/AlbumSearchCriteria.cs

```diff
@@ -7,12 +8,13 @@ public class AlbumSearchCriteria : SearchCriteriaBase
 
         public string AlbumTitle { get; set; }
         public int AlbumYear { get; set; }
+        public string Disambiguation { get; set; }
 
-        public string AlbumQuery => GetQueryTitle(AlbumTitle);
+        public string AlbumQuery => GetQueryTitle(AlbumTitle+(Disambiguation.IsNullOrWhiteSpace() ? string.Empty : $"+{Disambiguation}"));
```

Comment:
Can you change this to string interpolation like you have the other string.

Something like:
`$"{AlbumTitle}{(Disambiguation.IsNullOrWhiteSpace() ? string.Empty : $"+{Disambiguation}")}"`

Should be good to merge then